lib/api/controllers/writeController.js

```diff
@@ -31,6 +31,9 @@ module.exports = function WriteController (kuzzle) {
         if (rooms.length > 0) {
           kuzzle.notifier.notify(rooms, new ResponseObject(requestObject).toJson());
         }
+        else if (!requestObject.isPersistent()) {
+          kuzzle.notifier.notify({}, new ResponseObject(requestObject), context.connection);
+        }
```

Comment:
Indeed, this is not related to the authorization. 

This was added to cover the case where a REST non-persistent write call would not give any response but I am not sure the current implementation is the proper one, especially regarding how it will react with a webSocket connection.

Additionally, it may indeed be worth moving it into another PR.

I am checking how it can be improved/fixed.